osbs/build/user_params.py

```diff
@@ -126,14 +124,10 @@ def set_params(self,
         self.koji_parent_build.value = koji_parent_build
         self.koji_upload_dir.value = koji_upload_dir
         self.flatpak.value = flatpak
-        self.flatpak_base_image.value = flatpak_base_image
```

Comment:
You'll need to change atomic_reactor/schemas/user_params.json in atomic-reactor

Reply:
ok will make that change